javascript/ember/commands/run

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eu
+
+(
+  echo "Install, link and build integration"
+  cd /integration
+  script/setup
+  mono bootstrap
+  mono clean
+  mono build
```

Comment:
This takes forever on my laptop to build all packages. I suggest only building the packages we actually need.

```suggestion
  mono build --package=@appsignal/types,@appsignal/core,@appsignal/javascript,@appsignal/ember
```

Reply:
I'll do a separate PR applying this change to this and other packages.